fv3core/stencils/a2b_ord4.py

```diff
@@ -517,47 +553,47 @@ def __call__(self, qin: FloatField, qout: FloatField):
         self._sw_corner_stencil(
             qin,
             qout,
-            self.grid.agrid1,
-            self.grid.agrid2,
-            self.grid.bgrid1,
-            self.grid.bgrid2,
+            self._agrid1,
+            self._agrid2,
+            self._bgrid1,
+            self._bgrid2,
```

Comment:
What I'd really like to do is make these "compile time constants", i.e. even if they aren't actually compiled into the generated code, pass these in when we define the FrozenStencil and then have them be automatically included as extra call arguments. If we do this, I think we don't need to store them as attributes at all.

Reply:
If we figure out that the grid data can be initialized with very few free configuration settings I'm not opposed to making the metric term container object sooner rather than later, like we did for GridIndexing. Requires figuring out what's needed there, though.